ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,405 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Delegate protocol to handle user interaction with the CardView
+@objc(MSFCardDelegate)
+public protocol CardDelegate {
+    /// Called after the Card or the button inside the Card is tapped
+    /// - Parameter card: The Card that was tapped
+    @objc optional func didTapCard(_ card: CardView)
+}
+
+/// Color theme can be app colors, neutral colors, or custom colors. The theme affects the background, border, title, subtitle and icon colors
+@objc(MSFCardColorTheme)
+public enum CardColorTheme: Int, CaseIterable {
```

Comment:
nit: theme means something very different in Fluent design system. do "style" maybe better?